### ISSUE TITLE
Possible Checksum & Metadata Issue

### ISSUE THREAD

**Issue by DrizzleBoX** [Tenpi/soundcloud-download#24]
I'm getting a couple songs that are missing meta data and are skipping small segments of the song.
This occurs on both Chrome and Firefox.
I also confirm this issue is consistent across multiple devices.

Examples:
https://soundcloud.com/dallask/looking-for-your-love
https://soundcloud.com/mau5trap/deadmau5-imaginary-friends-i_o-remix
https://soundcloud.com/notdofficial/start-it-over-feat-cvbz-shy
https://soundcloud.com/navosmusicuk/believe-me
https://soundcloud.com/dabilliondollarbaby/blind-feat-young-thug

About 10% of the songs I download have this issue.
Let me know if you require additional examples.

**Comment by Tenpi** [comment]
it's intentional, a small portion of tracks are being downloaded with third party library [crunker.js](https://github.com/jackedgson/crunker), which doesn't add any metadata

**Comment by DrizzleBoX** [comment]
> it's intentional, a small portion of tracks are being downloaded with third party library [crunker.js](https://github.com/jackedgson/crunker), which doesn't add any metadata

What about the songs cutting out every few seconds?

**Comment by Tenpi** [comment]
> What about the songs cutting out every few seconds? Crunker also causing that?

Yes, it seems like Crunker doesn't do a seamless audio concatenation and there is a tiny pause between each of the audio segments, but at the very least it works. 

In the future I might find something else that's better, but that's what I have for now.
